src/fragments/lib/auth/js/manageusers.mdx

```diff
@@ -115,6 +115,24 @@ You can also access the user's attributes like their email address, phone number
 const { attributes } = await Auth.currentAuthenticatedUser();
 ```
 
+<Callout info>
```

Comment:
To use markdown links in the callout component you will need to add empty lines between the callout tags and remove all other tags within the callout component and use markdown syntax for all of them

```
<Callout info>

  Note: In order to retrieve attributes for federated users, the 
  **aws.cognito.signin.user.admin** scope must be included when
  configuring the Auth resource with an OAuth flow enabled.  
  &nbsp;  
  This scope grants access to Amazon Cognito User Pool API operations that require access tokens such as
  [GetUser](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_GetUser.html), 
  [UpdateUserAttributes](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_UpdateUserAttributes.html), 
  and [VerifyUserAttribute](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_VerifyUserAttribute.html)

</Callout>
```
The &nbsp; line is to create multiple line breaks in a row, normally markdown will combine those line breaks (markdown syntax for a line break is to end a line in 2 spaces) but by using the space encoded key you can put in multiple line breaks that will render as empty lines.